packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react';
+import { useAccountHistoryChart } from '@notional-finance/notionable-hooks';
+import { FiatSymbols } from '@notional-finance/core-entities';
+import { useFiat } from '@notional-finance/notionable-hooks';
+import {
+  THEME_VARIANTS,
+  SECONDS_IN_MONTH,
+  getNowSeconds,
+} from '@notional-finance/util';
+import { colors } from '@notional-finance/styles';
+import { useThemeVariant } from '@notional-finance/notionable-hooks';
+import { FormattedMessage } from 'react-intl';
+
+export const useTotalsChart = () => {
+  const baseCurrency = useFiat();
+  const [secondsMultiple, setSecondsMultiple] = useState(1.5);
+
+  window.addEventListener('resize', () => {
```

Comment:
Should this go inside a useEffect? It looks like a new listener will get added on every render

Reply:
If you put this in a useEffect then the window data points will not display as the screen gets smaller and only if a user refreshes on the smaller screen size.

Reply:
If you're fine with that functionality I can update it

Reply:
Can't you do this?
https://stackoverflow.com/questions/36862334/get-viewport-window-height-in-reactjs